ibm/mas_devops/roles/manage_mmi_monitoring_configure/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
This readme must be completed before this is merged.  Provide details about what this role is doing and why it would be used/what a user can do after they do this, e.g. explain basically what MMI monitoring is (what does MMI stand for), what it gives the user that they didn't have before they run this role, what they need to do to see the results of this monitoring (e.g. is there a dashboard somewhere, how do they find out the URL of the dashboard) etc

Reply:
README is updated.